lib/node_modules/@stdlib/math/base/complex/divide/test/test.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var cdiv = require( './../lib' );
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof cdiv, 'function', 'main export is a function' );
+	t.end();
+});
+
+function testCdiv( a, b, c, d, e, f, label ) {
+	tape(label, function test( t ) {
+		var expected;
+		var v;
+
+		v = cdiv( a, b, c, d );
+		expected = [ e, f ];
+
+		t.deepEqual( v, expected, '( ' + a + ' + ' + b + 'j ) / ( ' + c + ' + ' + d + 'j ) == ' + e + ' + ' + f + 'j' );
+
+		t.end();
+	});
+}
+
+testCdiv( -13.0, -1.0, -2.0, 1.0, 5.0, 3.0, 'simple division problem' );
+testCdiv( 1e308, 5e307, 1.0, 0.5, 1e308, 0, 'ab >= LARGE_THRESHOLD' );
```

Comment:
For floating-point numbers, we always include the decimal, even in scientific notation; e.g., `1.0e308`.